src/typing_extensions.py

```diff
@@ -1189,7 +1188,7 @@ class Point2D(TypedDict):
             ) + example + "."
             warnings.warn(deprecation_msg, DeprecationWarning, stacklevel=2)
             # Support a field called "closed"
-            if closed is not False and closed is not True:
+            if closed not in (False, True, None):
```

Comment:
This works differently for things that `==` equal True/False (such as 0/1):

```suggestion
            if closed is not False and closed is not True and closed is not None:
```